src/main/java/org/killbill/billing/plugin/payment/retries/api/ErrorMessage.java

```diff
@@ -17,15 +17,45 @@
 
 package org.killbill.billing.plugin.payment.retries.api;
 
-public enum ErrorMessage {
-
-    ADDRESS_MISMATCH,
-    CARD_NUMBER_MISMATCH,
-    CVV_MISMATCH,
-    EXPIRATION_DATE_MISMATCH,
-    EXPIRED_CARD,
-    FRAUD,
-    GENERAL_DECLINE,
-    INSUFFICIENT_FUNDS,
-    LOST_OR_STOLEN
+import java.util.Arrays;
+import java.util.function.Predicate;
+
+import com.google.common.base.Strings;
+
+public enum ErrorMessage implements Predicate<String> {
+
+    ADDRESS_MISMATCH(false, "address"),
+    CARD_NUMBER_MISMATCH(false, "invalid card number"),
+    CVV_MISMATCH(false, "cvc"),
+    EXPIRATION_DATE_MISMATCH(false),
```

Comment:
Missing keywords?

Reply:
Actually, I am not sure what would a suitable keywords for this category.

Reply:
Adyen returns `101 : Expired Card / Invalid Expiration Date` for instance, maybe grep for `expir[ed|ation]`?

Reply:
For `EXPIRATION_DATE_MISMATCH`, I added keyword: `expiration date`. 
Also added `100: Deny` for Adyen.